src/QuokkaSimulation.hpp

```diff
@@ -171,8 +177,19 @@ template <typename problem_t> class QuokkaSimulation : public AMRSimulation<prob
 
 	explicit QuokkaSimulation(amrex::Vector<amrex::BCRec> &BCs_cc) : AMRSimulation<problem_t>(BCs_cc) { initialize(); }
 
-	inline void initialize()
+	void initialize()
 	{
+		static_assert(!(Physics_Traits<problem_t>::is_mhd_enabled && Physics_Traits<problem_t>::is_radiation_enabled),
+			      "MHD + Radiation is not supported yet.");
```

Comment:
Wait, why is MHD + radiation not supported?

Reply:
It's completely untested. I am almost certain that there exist several places in the radiation code where Eint is computed as Etot - KE, which is wrong when Emag != 0.

Reply:
OK, that's fine. This is obviously on the to-do list, then.